Packs/CyberArkAIMv2/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py

```diff
@@ -0,0 +1,166 @@
+            kf.flush()
+            return (cf.name, kf.name), cf, kf
+
+    def list_credentials(self):
+        url_suffix = f'/AIMWebService/api/Accounts?AppID={self._app_id}&Safe=' \
+                     f'{self._safe}&Folder={self._folder}&Object={self._credential_object}'
+        params = {
+            "AppID": self._app_id,
+            "Safe": self._safe,
+            "Folder": self._folder,
+            "Object": self._credential_object,
+        }
+
+        return self._http_request("GET", url_suffix, params=params, auth=self.auth, cert=self.crt)
+
+
+def list_credentials_command(client):
+    """Lists all credentials available.
+    :param client: the client object with the given params
+    :return: the credentials info without the explicit password
+    """
+    res = client.list_credentials()
+    demisto.log(str(res))
+    # the password value in the json appears under the key "Content"
+    if "Content" in res:
+        del res["Content"]
+    # notice that the raw_response doesn't contain the password either
+    results = CommandResults(
+        outputs=res,
+        raw_response=res,
+        outputs_prefix='CyberArkAIM',
+        outputs_key_field='Name',
+    )
+    return results
+
+
+def fetch_credentials(client):
+    """Fetches the available credentials.
+    :param client: the client object with the given params
+    :return: a credentials object
+    """
+    res = client.list_credentials()
+
+    credentials = {
+        "user": res.get("UserName"),
+        "password": res.get("Content"),
+        "name": res.get("Name"),
+    }
+    demisto.credentials([credentials])
+
+
+def test_module(client: Client) -> str:
+    """Performing a request to the AIM server with the given params
+    :param client: the client object with the given params
+    :return: ok if the request succeeded
+    """
+    client.list_credentials()
+    return "ok"
+
+
+def main():
+
+    params = demisto.params()
+
+    url = params.get('url', "")
+    use_ssl = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+
+    app_id = params.get('app_id', "")
+    folder = params.get('folder', "")
+    safe = params.get('safe', "")
+    credential_object = params.get('credential_names', "")
+
+    cert_text = params.get('cert_text', "")
+    key_text = params.get('key_text', "")
+
+    username = ""
+    password = ""
+    if params.get('credentials'):
+        # credentials are not mandatory in this integration
+        username = params.get('credentials').get('identifier')
+        password = params.get('credentials').get('password')
+
+    try:
+        client = Client(server_url=url, use_ssl=use_ssl, proxy=proxy, app_id=app_id, folder=folder, safe=safe,
+                        credential_object=credential_object, username=username, password=password,
+                        cert_text=cert_text, key_text=key_text)
+
+        command = demisto.command()
+        LOG(f'Command being called in CyberArk AIM is: {command}')
```

Comment:
`LOG` is being deprecated. It is suggested to use `demisto.debug` directly instead.